src/sentry/experiments/manager.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+import six
+
+
+class ExperimentManager(object):
```

Comment:
add a comment that this experiment stuff is in sentry because we want experiment assignments to be available on the frontend included in the serialized org details, and the org serializer is in sentry. it's non-obvious why this is not in getsentry.